genisys/main.py

```diff
@@ -71,6 +73,8 @@ def run(subcommand, args):
         generate_config(yaml_parser, args.root)
     elif subcommand == "server":
         genisys.server.run(yaml_parser)
+    elif subcommand == "password":
+        MakePassword.make_password(mkpass, args.password, args.file)
```

Comment:
I had to check this myself becaue I've not seen a method called this way before. Interesting that it works! Since it doesn't impact functionality, a change is not needed here. Just an FYI that normally instead of doing `Type.method(instance, ...args)` you do `instance.method(...args)`

Reply:
Right I normally do it based on the instance I actually just forgot to change that since it did not throw an error. I will change it anyway for best practices.